index.js

```diff
@@ -11,16 +11,19 @@ const eventPath = process.env.GITHUB_EVENT_PATH;
 const runId = process.env.GITHUB_RUN_ID;
 
 const readEventFile = () => fs.readFileSync(eventPath, 'utf8');
-const commitMessage = () => JSON.parse(readEventFile()).commits[0].message;
+const escapeUnicode = (str) => str.replace(/[^\0-~]/g, (ch) => 
```

Comment:
Why can't we use this function in line 83/84?
```
const data = replacer(inputSlackJson);
const result = await sendMessage(escapeUnicode(data));
```

Reply:
Done ✅